### PR TITLE
NEW: TPT3-88. Client Service Authorization

### PR DESCRIPTION
**Pull request by denys-davydov**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,7 +34,7 @@ project.ext {
     projectVersion = projectMajorVersion + '-SNAPSHOT'
 //    projectVersion = projectMajorVersion
     dropwizardVersion = '1.1.0'
-    coreApiVersion = '0.5.3_134'
+    coreApiVersion = '0.5.2-SNAPSHOT'
```
Comment (AlexanderSerbin) [inline build.gradle]: We agreed not to use SNAPSHOT version when committing to GitHub. Use SNAPSHOT versions for local purpose only

Comment (denys-davydov) [reply build.gradle]: Thank you Alex. After the api-core project pull request is merged and the is successful I will update the dependency version here.

**Comment by denys-davydov** [comment]
Please do not merge until this is done
https://github.com/ca-cwds/cals-api/pull/286#discussion_r158868682

File: build.gradle (hunk 1 above)
Comment (denys-davydov) [reply build.gradle]: @AlexanderSerbin It was updated, please review

**Pull request merged**